src/app/settings.py

```
@@ -397,3 +397,9 @@ def show_toolbar(request):
     "AWS_REGION": AWS_REGION,
     "EXTRACTORS_CACHE_DIR": "/tmp/classifier",
 }
+
+# Reporting S3 credentials
+REPORT_S3_ACCESS_KEY_ID = os.environ.get("REPORT_S3_ACCESS_KEY_ID")
+REPORT_S3_SECRET_ACCESS_KEY = os.environ.get("REPORT_S3_SECRET_ACCESS_KEY")
+# if not REPORT_S3_ACCESS_KEY_ID or not REPORT_S3_SECRET_ACCESS_KEY:
+#     raise ValueError("REPORT_S3_ACCESS_KEY_ID and REPORT_S3_SECRET_ACCESS_KEY must be set.")
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)